src/adapters/MeetingsJSONAdapter.js

```diff
@@ -66,14 +71,108 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       observer.complete();
     });
 
-    return getMeeting$.pipe(
+    // Attach local media stream if meeting `localVideo` property is not `null`.
+    // We can not attach the MediaStream object in a JSON module, the work needs
+    // to be done here.
+    const getMeetingWithLocalMedia$ = getMeeting$.pipe(
       flatMap((meeting) =>
-        // Attach the localMedia stream if meeting localVideo
-        // property is not `null`. Since we can not attach the
-        // MediaStream object in out JSON module, the work needs
-        // to be done here.
         from(this.getLocalVideo()).pipe(map((localVideo) => (meeting.localVideo ? {...meeting, localVideo} : meeting)))
       )
     );
+
+    // Send updates on the meeting when a mute event is triggered
+    const muteEvent$ = fromEvent(document, MUTE_AUDIO_CONTROL).pipe(
+      filter((event) => event.detail.ID === ID),
+      map((event) => event.detail)
+    );
+
+    return concat(getMeetingWithLocalMedia$, muteEvent$);
+  }
+
+  /**
+   * Returns a MediaStream object obtained from local user media.
+   * @returns {MediaStream}
+   */
+  async getLocalVideo() {
+    const constraints = {
+      video: true,
+      audio: false,
+    };
+    let stream;
+
+    try {
+      stream = await navigator.mediaDevices.getUserMedia(constraints);
+    } catch (reason) {
+      // eslint-disable-next-line no-console
+      console.error('Meetings JSON adapter can not display the user local stream', reason);
+    }
+
+    return stream;
+  }
+
+  /**
+   * Returns an observable that emits the display data of a meeting control.
+   *
+   * @param {string} ID ID of the meeting for which to update display
+   * @returns {Observable<MeetingControlDisplay>}
+   * @memberof MeetingJSONAdapter
+   * @private
+   */
+  muteAudioControl(ID) {
+    const unmuted = {
+      ID: MUTE_AUDIO_CONTROL,
+      icon: 'microphone',
+      tooltip: 'Mute',
+      state: MeetingControlState.INACTIVE,
+    };
+    const muted = {
+      ID: MUTE_AUDIO_CONTROL,
+      icon: 'microphone-muted',
+      tooltip: 'Unmute',
+      state: MeetingControlState.ACTIVE,
+    };
+
+    const default$ = Observable.create((observer) => {
+      const meeting = this.datasource[ID];
+
+      if (meeting && meeting.localAudio) {
```

Comment:
I think if the meeting is not defined, we should throw an error and to mix it up with `meeting.localAudio` check.